docs/modules/data-structures/pages/listening-for-map-entries.adoc

```diff
@@ -171,7 +198,7 @@ When the listener is running, a change to any record with the name `smith` will
 
 ```
 entryAdded:EntryEvent {Address[192.168.178.10]:5702} key=1,oldValue=null,
-value=Person{name= smith }, event=ADDED, by Member [192.168.178.10]:5702
+value={surname=smith}, event=ADDED, by Member [192.168.178.10]:5702
```

Comment:
why is it changed?

Reply:
there is no object called Person in the code, I think it is a mistale